python/lsst/obs/pfs/pfsMapper.py

```diff
@@ -200,6 +200,16 @@ def std_raw(self, item, dataId):
         does not exist.
 
         See _shiftAmpPixels() for the implementation.
+
+        Parameters
+        ----------
+        item : image-like object
+            Can be any of lsst.afw.image.Exposure,
+            lsst.afw.image.DecoratedImage, lsst.afw.image.Image
+            or lsst.afw.image.MaskedImage
```

Comment:
You've described the type. Here, you need to say _what_ it is.

Reply:
Done.